### PR TITLE
Move to GitHub Actions and Setup Auto Release

### PR DESCRIPTION
**Pull request by svanherk**
Needed to add a new constant and figured I'd move off Travis and automate the full release process while here.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          persist-credentials: false
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version-file: .nvmrc
+          registry-url: 'https://registry.npmjs.org'
```
Comment (svanherk) [inline .github/workflows/release.yml:20]: Couldn't find an exact example of npm publishing plus incremental-release, so I tried to go based on those reverted changes in the action - @dlockhart does this look good?

File: package.json [hunk 2]
```diff
@@ -14,6 +14,11 @@
   "version": "6.67.0",
   "main": "dist/index.js",
   "module": "index.js",
+  "files": [
```
Comment (svanherk) [inline package.json:17]: Not totally sure how `npm` is grabbing the correct files currently without this, maybe it's using `module and `main` and anything that includes those files?  Figured adding this was clearer and safer

File: .github/workflows/release.yml (hunk 1 above)
Comment (dlockhart) [reply .github/workflows/release.yml:20]: Yep looks correct!

File: package.json (hunk 2 above)
Comment (dlockhart) [reply package.json:17]: Yep seems like it!

> Certain files are always included, regardless of settings:
package.json
README
LICENSE / LICENCE
The file in the "main" field

**Review by dlockhart — APPROVED** [review]

**Pull request merged**